### PR TITLE
Address pr 646 comments

### PR DESCRIPTION
**Pull request by LucasRoesler**
## Description

Address several comments from #646 

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

All of the changes are very minor, unit tests pass

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by alexellis** [comment]
Thoughts on this one? https://github.com/openfaas/faas-netes/pull/646#pullrequestreview-449020895

**Comment by alexellis** [comment]
Would be nice to have these back https://github.com/openfaas/faas-netes/pull/646/files#diff-7ddfb3e035b42cd70649cc33393fe32cL91-L94

**Comment by LucasRoesler** [comment]
> Would be nice to have these back https://github.com/openfaas/faas-netes/pull/646/files#diff-7ddfb3e035b42cd70649cc33393fe32cL91-L94

I don't understand what you mean, those were not removed. The ClusterRole should have access to the Profiles

File: chart/openfaas/templates/controller-rbac.yaml [hunk 1]
```diff
@@ -179,12 +212,12 @@ metadata:
     component: faas-controller
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
-  name: {{ .Release.Name }}-controller
-  namespace: {{ $functionNs | quote }}
+  name: {{ .Release.Name }}-profiles
```
Comment (LucasRoesler) [inline chart/openfaas/templates/controller-rbac.yaml:215]: note that this diff is misleading, i have only added a new Role and RoleBinding, this is just groupnig the changes in an odd way to make it look like i edited the existing RoleBinding

Comment (alexellis) [reply chart/openfaas/templates/controller-rbac.yaml:215]: Thanks for that note 👍

**Comment by alexellis** [comment]
I can see why you're confused, I think the lines changed or didn't copy.

Screenshot 2020-07-15 at 17 33 34

**Comment by LucasRoesler** [comment]
> I can see why you're confused, I think the lines changed or didn't copy.
> 
> Screenshot 2020-07-15 at 17 33 34

👍  i added this https://github.com/openfaas/faas-netes/pull/669/commits/9a2b3d3ac1474aa9d0aeea615c6191db97505818 to ensure it will be pretty printed now

**Comment by alexellis** [comment]
This needs to be marked as non-draft to be able to be merged.

There is a rebase error in `yaml/` too

**Pull request merged**